uswds/uswds-compile
# USWDS-Compile - POAM: June '24

# Summary

POAM for June 2024.

Cleared dependency cache and rebuilt.

## Dependency changes:

**Before:**
```jsx
4 vulnerabilities (1 low, 3 moderate)
```

**After:**

```jsx
3 moderate severity vulnerabilities
```

## Testing instructions

1. Install this branch on site
    - On Site's `main` branch, run: 
  ```bash
npm install "https://github.com/uswds/uswds-compile/tree/cm-POAM-june-2024" --save
```
2. Run gulp sass scripts and ensure compile runs as expected without error.
3. Ensure no when package is used to compile site


## Dependency updates

No direct dependency updates.
